varnish-sys/build.rs

```diff
@@ -112,6 +113,9 @@ fn find_include_dir(out_path: &PathBuf) -> Option<(Vec<PathBuf>, String)> {
             if major == 7 && minor <= 5 {
                 println!("cargo::rustc-cfg=feature=\"_objcore_in_init\"");
             }
+            if major == 6 && minor <= 5 {
```

Comment:
should the `minor` be checked here?  Seems like it should only test for `major == 6` ?

Reply:
copy-pasta, we really only want to support `minor == 0` here